src/legacy/core_plugins/kibana/public/management/sections/objects/view/components/form.tsx

```diff
@@ -173,6 +180,53 @@ export const Form = ({ initialFields, onChange, canEdit }: Props) => {
           </EuiFormRow>
         );
       })}
+      <EuiFormRow fullWidth={true}>
+        <EuiFlexGroup>
+          {canEdit ? (
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                fill={true}
+                aria-label={i18n.translate('kbn.management.objects.view.saveButtonAriaLabel', {
+                  defaultMessage: 'Save { title } Object',
+                  values: { title },
+                })}
+                onClick={() => submit()}
+                disabled={fields
+                  .filter(f => f.type === 'json' || f.type === 'array')
+                  .map(f => {
+                    try {
+                      JSON.parse(f.value || '[]');
+                      return true;
+                    } catch (e) {
+                      return false;
+                    }
+                  })
+                  .includes(false)}
```

Comment:
This is the validation part. 

`|| '[]'` in `f.value || '[]'` is there because `JSON.parse(undefined)` or `JSON.parse('')` is error.